src/components/create_cluster/index.js

```diff
@@ -441,25 +422,50 @@ class CreateCluster extends React.Component {
                     );
                   }
                 })}
-                <div
-                  className={
-                    'col-4 new-cluster--add-worker-button ' +
-                    (this.state.workers.length < 3 ? 'warning' : '')
-                  }
-                  onClick={this.addWorker}
+              </div>
+            </div>
+
+            <div className='row section'>
+              <div className='col-3'>
+                <h3 className='table-label'>Node Count</h3>
+              </div>
+              <div className='col-9'>
+                <form
+                  onSubmit={e => {
+                    e.preventDefault();
+                  }}
                 >
-                  <div className='new-cluster--add-worker-button-title'>
-                    Add a worker
-                  </div>
-                  {this.state.workers.length < 3 ? (
-                    <div className='new-cluster--low-worker-warning'>
-                      We recommend that you have at least three worker nodes in
-                      a cluster
+                  <div>
+                    <p>
+                      To have full control over cluster scaling, set both
+                      numbers to the same value.
+                    </p>
+                    <div className='col-3'>
+                      <p>Minimum number of nodes.</p>
```

Comment:
```suggestion
                      <p>Minimum</p>
```

Reply:
A label doesn't need punctuation. Also truncated it to remove duplication with the section label.

Reply:
You may notice that the implementation is based on the mockups we agreed on.